### PR TITLE
Update for 2022.2

### PR DESCRIPTION
**Pull request by fbouliane**
Hello,
Not sure if this is welcome or not. 
I did a couple modifications to be able to build it for 2022.1.3.

I don't have an idea of the impacts of the version bump.
took some changes from https://github.com/JetBrains/intellij-platform-plugin-template

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -5,18 +5,18 @@ pluginGroup = com.github.nokia.pyvenv
 pluginName_ = PyVenv Manage
 pluginVersion = 1.3.5
 pluginSinceBuild = 193
-pluginUntilBuild = 213.*
+pluginUntilBuild = 231.*
```
Comment (fbouliane) [inline gradle.properties:8]: probably too wide

**Comment by jwheeler31** [comment]
This PR is over a month old. Is there anything specific holding it up?

**Comment by fbouliane** [comment]
I would have expected some feedback from one of the core contributors
It works when I build it locally for my personal usage. 

@gaborbernat 
@CsatariGergely 
or
@andrask

**Comment by gaborbernat** [comment]
As far as I know the maintainer left nokia, so this project is now mostly abandoned.

**Comment by gaborbernat** [comment]
I did create a fork here https://github.com/gaborbernat/PyVenvManage I've been using myself, but don't have the bandwith to publish it publicly.

**Comment by andrask** [comment]
Dear @gaborbernat and dear @fbouliane, it took me too long to figure out how we could continue this project. I got approval to move it to an independent organization, so the project lives on here https://github.com/pyvenvmanage/PyVenvManage. I will resume the releasing and publishing actions too.

I would highly appreciate your contributions. I'd like to avoid copying and pasting your commits with references to your original works.

**Comment by gaborbernat** [comment]
Do you have the rights to publish to the pycharm plugin repository so we can continue under the current name?

**Comment by gaborbernat** [comment]
I did add my changes here https://github.com/pyvenvmanage/PyVenvManage/pull/1, so we just need to setup GHA publishing to pycharm and we should be good to go.

**Pull request closed without merging**